sympy/functions/elementary/complexes.py

```diff
@@ -754,6 +852,35 @@ def _eval_is_algebraic(self):
 class transpose(Function):
     """
     Linear map transposition.
+
+    Examples
+    ========
+
+    >>> from sympy.functions import transpose
+    >>> from sympy.matrices import MatrixSymbol
+    >>> A = MatrixSymbol('A', 25, 9)
+    >>> transpose(A)
+    A.T
+    >>> B = MatrixSymbol('B', 9, 22)
+    >>> transpose(B)
+    B.T
+    >>> transpose(A*B)
+    B.T*A.T
+
+    Parameters
+    ==========
+
+    arg : MatrixSymbol
+        Symbolic representation of a Matrix Object.
```

Comment:
I think that more survey should be done for this function because it is used in `tensor` or `quantum`.
And it is applied more generally than `MatrixSymbol` at least, e.g. `MatrixExpr`

Reply:
I will look it up and also ask in the gitter room.